notebooks/Exp Gradient for DP(adultl).ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
Make this a separate PR. The notebook should have more than just the code--it needs to provide the motivation, e.g., what does it try to accomplish? And it needs to explain the steps in it more carefully.